inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -140,6 +141,43 @@ ModelPtr FrontEnd::buildInitialModel(ie::ICNNNetwork& network) {
     return runCommonPasses(network);
 }
 
+bool FrontEnd::isLayerSupported(const std::string& type) {
+    return parsers.count(type) != 0;
+}
+
+ie::ICNNNetwork::Ptr FrontEnd::convertNetwork(ie::ICNNNetwork& network) {
+        std::shared_ptr<ie::ICNNNetwork> convertedNetwork;
+        // disable GeLU decomposition
+        const auto transformationsPredicate = [](const std::shared_ptr<const ngraph::Node> &node) -> bool {
```

Comment:
Please, connect with @AndrewBakalinIntel and double check this place according with #2041

Reply:
#2041 was merged. Please, resolve conflicts if any

Reply:
Done